spec/acceptance/suites/default/10_pki_spec.rb

```diff
@@ -10,27 +10,31 @@
   end
 
   hosts_with_role(hosts, 'directory_server').each do |host|
-    let(:ds_root_name) do
-      'puppet_default'
-    end
-    let(:root_dn_password_file) do
-      "/usr/share/puppet_ds389_config/#{ds_root_name}_ds_pw.txt"
-    end
+    context "on #{host} " do
+      let(:ds_root_name) { 'test_tls' }
+      let(:base_dn) { 'dc=tls,dc=com' }
+      let(:rootpasswd) { 'password'}
+      let(:root_dn) { 'cn=Directory_Manager' }
+      let(:bootstrapldif) { ERB.new(File.read(File.expand_path('files/bootstrap.ldif.erb',File.dirname(__FILE__)))).result(binding) }
+      let(:fqdn) {  fact_on(host, 'fqdn').strip }
+      let(:certdir) {'/etc/pki/simp-testing/pki'}
 
-    fqdn = fact_on(host, 'fqdn').strip
-
-    context 'when enabling PKI in the default instance' do
+      let(:root_dn_password_file) do
```

Comment:
I don't think this is used.  Please remove.

Reply:
I removed it.